deps_proto/BUILD.bazel

```diff
@@ -7,14 +7,6 @@ go_proto_checkedin_test(
     src = "deps.gen.pb.go",
 )
 
-genrule(
-    name = "copy_and_fix",
-    srcs = ["@io_bazel//src/main/protobuf:srcs"],
-    outs = ["deps.proto"],
-    cmd = ("SRCS=($(locations @io_bazel//src/main/protobuf:srcs));" +
-           "cp $$(dirname $$SRCS)/deps.proto $@"),
-)
-
 proto_library(
     name = "blaze_deps_proto",
     srcs = ["deps.proto"],
```

Comment:
same comments as below -- add version/commit as a comment, should we leave a variation on copy_and_fix that writes to 'deps.proto.vendor' and then a shell script that can be run to clone, copy and replace the vendored files?